.buildbot.sh

```diff
@@ -35,6 +35,23 @@ for dir in . employee shared_objects timsort; do
     popd
 done
 
+echo "Checking that all the hybrid examples build on Morello..."
+platform='morello-hybrid'
+for dir in hybrid; do
+    pushd "$dir"
+        make -f Makefile.$platform clean
+        if [ -d bin/$platform ]; then
```

Comment:
Is this check really necessary? It seems *very* paranoid!

Reply:
I have copied from the previous PR. I may agree with you, though.

Reply:
OK, let's kill these checks and then in another commit kill the other checks too?

Reply:
Agreed.

Reply:
Done in 49c7009.

Reply:
> OK, let's kill these checks and then in another commit kill the other checks too?

Sorry, I have just realised you said in another *commit* not PR. I had to kill the other checks too here, then...